low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -539,6 +541,55 @@ def process(self):
         logger.info(f"{self.name} interface not implemented.")
 
 
+class PreUpgradeCmd(Cmd):
+    """PreUpgrade support for SSPL component."""
+
+    name = "pre_upgrade"
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def validate(self):
+        # Common validator classes to check Cortx/system wide validator
+        pass
+
+    def process(self):
+        logger.info(f"Nothing to be done for {self.name}.")
+
+
+class PostUpgradeCmd(Cmd):
+    """PostUpgrade support for SSPL componenet."""
+
+    name = "post_upgrade"
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def validate(self):
+        # Common validator classes to check Cortx/system wide validator
+        pass
+
+    def process(self):
+        new_conf_url = 'yaml:///opt/seagate/cortx/sspl/conf/sspl.conf.LR2.yaml'
+        existing_conf_url = 'yaml:///etc/sspl.conf'
```

Comment:
Can we read config filename from sspl_constant.

Reply:
Done